Castlevania/GameMap.cpp

```diff
@@ -44,7 +44,21 @@ void CMap::LoadMap(LPCWSTR dataPath) {
 
 		while (ss >> n)
 		{
-			idTile = n + 1000;
+			switch (mapID)
+			{
+			case 1:
+				idTile = n + 1000;
+				break;
+			case 2: 
+				idTile = n + 2000;
+				break;
+			case 3:
+				idTile = n + 3000;
+				break;
```

Comment:
Avoid hardcoding numbers

Reply:
Yes, please define it on GameMap.h such as PREFIX_ID_TILE_MAP1 1000, PREFIX_ID_TILE_MAP1 2000, PREFIX_ID_TILE_MAP1 3000. Because I assign id for tiles in game map likes this 1001, 1002, 1003 for map 1, 2002, 2003 for map 2 and map 3 is also too.